CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+* Add support for managing workspace drift detection setting [#462](https://github.com/hashicorp/go-tfe/pull/462)
```

Comment:
It looks like this PR adds support for enforcing assessments in an organization and workspace. Could you update the changelog entry to reflect that?